shared/AppInsightsCommon/Tests/Unit/src/ThrottleMgr.tests.ts

```diff
@@ -225,7 +225,7 @@ export class ThrottleMgrTest extends AITestClass {
                     } as IThrottleLimit,
                     interval: {
                         monthInterval: 1,
-                        dayInterval: 31,
+                        dayInterval: 32,
```

Comment:
This should be set to 32 to make sure that it is invalid to trigger throttle any day.